Source/Core/Core/State.cpp

```diff
@@ -608,7 +613,7 @@ bool ReadHeader(const std::string& filename, StateHeader& header)
   std::lock_guard lk(s_save_thread_mutex);
 
   File::IOFile f(filename, "rb");
-  return ReadStateHeaderFromFile(header, f);
+  return ReadStateHeaderFromFile(header, f, false);
```

Comment:
The meaning of this parameter would be clearer if it was an enum class (ReadVersionString::No, perhaps?).

Reply:
C++ actually has a built-in type to distinguish between "yes" and "no" - it's called a boolean.

Maybe the parameter could be more descriptive, as a boolean called `version` could be interpreted many different ways.

Reply:
What I sometimes see people do to clear up ambiguous boolean usage in function calls is:
```C++
bool get_version_info = false;
return ReadStateHeaderFromFile(header, f, get_version_info);
```
If others agree this is better then I can apply it.

Reply:
Yeah anything to clarify would be better than just false.  I don't know if some would insist on an enum..

Reply:
The caller and the callee are in the same file and no 50 lines apart, I don't think this needs all that much clarification. But yeah I guess this is fine.